sound/soc/sof/ipc.c

```diff
@@ -175,6 +175,15 @@ static void ipc_log_header(struct device *dev, u8 *text, u32 cmd)
 		break;
 	case SOF_IPC_GLB_TRACE_MSG:
 		str = "GLB_TRACE_MSG"; break;
+	case SOF_IPC_GLB_TEST:
+		str = "GLB_TEST_MSG";
+		switch (type) {
+		case SOF_IPC_TEST_IPC_FLOOD:
+			str2 = "IPC_FLOOD"; break;
+		default:
+			str2 = "unknown type"; break;
+		}
+		break;
```

Comment:
shouldn't this only be allowed when SOF_DEBUG is enabled?

Reply:
@plbossart this is just the ipc message parser. It would never receive a flood type IPC message to parse when DEBUG is not enabled.